pytest/lib/cluster.py

```diff
@@ -1034,7 +1035,7 @@ def init_cluster(
     assert 0 == process.returncode, err
 
     node_dirs = [
-        line.split()[-1]
+        re.split('=|\s', line)[-1]
```

Comment:
This was required as we are reading the log to get the node_directory and the format of the log changed from just `{path}` to `node_dir={path}`